chromadb/telemetry/opentelemetry/__init__.py

```diff
@@ -146,9 +149,9 @@ def wrapper(*args: Any, **kwargs: Dict[Any, Any]) -> Any:
                 with tracer.start_as_current_span(trace_name, attributes=attributes):
                     return f(*args, **kwargs)
 
-            return wrapper
+            return wrapper  # type: ignore
 
-    return decorator
+    return decorator  # type: ignore
```

Comment:
figuring out the right type here was painful
needed to correctly propagate docstrings for async functions when decorated